kodilla-stream/src/main/java/com/kodilla/exception/homework/Warehouse.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.exception.homework;
+
+import java.util.*;
+
+public class Warehouse {
+    private List<Order> orders = new ArrayList<>();   //Lista zamówień
+
+    public Order addOrder(Order order) {   //Dodaje zamówienia do listy
+        orders.add(order);
+        return order;
+    }
+
+    public Order getOrder(String number) throws OrderDoesntExistException {  //Wyszukuje konkretne zamówienie z wyjątkiem
+        return orders
+                .stream()
+                .filter(order -> order.getNumber() == number)
+                .findFirst()
+                .orElseThrow(OrderDoesntExistException::new);
```

Comment:
Jest okej. Tylko jak dostaniesz taki wyjatek, to nie bedziesz wiedzial dlaczego. Warto dodac konstruktor do klasy wyjatku, ktory przyjmie stringa z numerem zamowienia.

public OrderNotFoubdExceprion(String orderNumber) {
    super(String.format(„Order with id %s does not exist”, orderNumber);
}